docroot/sites/iisc.uiowa.edu/modules/iisc_core/templates/views-view-unformatted--partners-recent.html.twig

```diff
@@ -0,0 +1,148 @@
+{#
```

Comment:
It seems like there are two different template comments here. We probably don't need either of these. Maybe just a reference to the base template that is being overridden.

Reply:
I removed the extra node template comments. I usually like having the template comment directly in the file, but just a reference to the base template doesn't add much of an extra step, so I'd be fine having it removed too.

Reply:
I'm fine with leaving it in there, if its helpful. I almost never refer to them (maybe I should!)